addons/MountMuzzle/README.md

```diff
@@ -0,0 +1,155 @@
+**Author:** [Sjshovan (Apogee)](https://github.com/Ap0gee)  
+**Version:** v0.9.2  
+
+
+# Mount Muzzle
+
+> A Windower 4 addon that allows the user to change or remove the default mount music in Final Fantasy 11 Online.
+
+
+### Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Commands](#commands)
+- [Support](#support)
+- [Change Log](#change-log)
+- [Known Issues](#known-issues)
+- [TODOs](#todos)
+- [License](#license)
+
+___
+### Prerequisites
+1. [Final Fantasy 11 Online](http://www.playonline.com/ff11us/index.shtml)
+2. [Windower 4](http://windower.net/)
+
+___
+### Installation
+1. Navigate to <https://github.com/Ap0gee/MountMuzzle>.
+2. Click on `Releases`. 
+3. Click on the `Source code (zip)` link within the latest release to download.
+4. Extract the zipped folder to `Windower4/addons/`.
+5. Rename the folder to remove the version tag (`-v0.9.2`). The folder should be named `MountMuzzle`.
+ 
+___
+### Usage
+
+**Manually load** the addon by using one of the following commands:
+    
+    //lua load mountmuzzle  
+    //lua l mountmuzzle
+    
+**Automatically load** this addon by adding `lua load mountmuzzle` or `lua l mountmuzzle` to the bottom of your `Windower4/scripts/init.txt` file.
```

Comment:
This is actually discouraged. Automatic loading should be handled either through the Windower launcher or through the `plugin_manager` addon, which offers finer grained control over character-based plugin and addon usage.

Reply:
Done.